lib/src/compiler-path.ts

```diff
@@ -5,8 +5,35 @@
 import * as fs from 'fs';
 import * as p from 'path';
 
+const {platform, arch} = process;
+
+const PACKAGES: {[key: string]: string} = {
+  'linux arm64': '@sass-embedded/linux-arm64',
+  'linux ia32': '@sass-embedded/linux-ia32',
+  'linux x64': '@sass-embedded/linux-x64',
+  'darwin arm64': '@sass-embedded/macos-arm64',
+  'darwin x64': '@sass-embedded/macos-x64',
+  'win32 ia32': '@sass-embedded/windows-ia32',
+  'win32 x64': '@sass-embedded/windows-x64',
+};
+
 /** The path to the embedded compiler executable. */
 export const compilerPath = (() => {
+  try {
+    const executable = require.resolve(
+      `${
+        PACKAGES[`${platform} ${arch}`]
```

Comment:
Nit: I think referring directly to `process.platform` and `process.arch` here would be cleaner.